microshift_updating/microshift-about-updates.adoc

```diff
@@ -6,11 +6,11 @@ include::_attributes/attributes-microshift.adoc[]
 
 toc::[]
 
-Updates are supported on {product-title} beginning with the General Availability version 4.14. With the 4.16 release, the following updates are supported:
+Updates are supported on {product-title} beginning with the General Availability version 4.14. With the 4.17 release, the following updates are supported:
 
 * A maximum of two minor versions to the next in sequence, for example, from 4.14 to 4.16.
-* One minor version to the next in sequence, for example, from 4.15 to 4.16.
-* Patch updates are also supported from z-stream to z-stream, for example 4.16.1 to 4.16.2.
+* One minor version to the next in sequence, for example, from 4.16 to 4.17.
```

Comment:
You might want to consider making a new variable, such as `ocp-version-n1` (similar to [this in the `common-attributes` file](https://github.com/openshift/openshift-docs/blob/main/_attributes/common-attributes.adoc#L207-L209)). If you create an attribute for the previous version of OCP/MicroShift/etc, you could update these version numbers once in the attributes file and be done with it.

Definitely not a blocker, but something to consider if you want to simplify the maintenance of these example versions.

Reply:
will check if this can be done. Thank you for the suggestion